contributors/devel/sig-architecture/staging.md

```diff
@@ -19,10 +20,24 @@ of the following staging repos in the k8s.io org:
 - apimachinery
 - apiserver
 - client-go
+- cli-runtime
```

Comment:
It's very likely that this list can get diverged easily when new repos are added.

I'd remove this list and just link to the staging README (linked above) since that's the source of truth.